src/GameController.cs

```diff
@@ -9,51 +9,16 @@
 PlayerController player = new(20);
 CommandController commandController = new(player, world);
 
-ConsoleController.WriteText("Welcome to ProjectFishStick!");
+ConsoleController.WriteText("Welcome to ProjectFishStick!\n");
 
+// Initial scene description before we begin the main gameplay loop
+ConsoleController.DescribeScene(world.GetScene(player.GetCurrentSceneId()));

```

Comment:
DescribeScene is outside of main game loop. New locations won't get displayed after moving.